crates/db_views/src/post_view.rs

```diff
@@ -386,6 +386,22 @@ fn queries<'a>() -> Queries<
 
     let (limit, offset) = limit_and_offset(options.page, options.limit)?;
 
+    // when_before and when_after to filter posts by specified published date.
+    // ToDo: establish default convention, likley specific to the sort choice picked
+    // For "Top*" sorting, this allows custom range, overrides filtering that sort choice previously set
+    if let Some(when_before) = options.when_before {
+      let parsed_when_before = chrono::NaiveDateTime::from_timestamp_millis(when_before);
```

Comment:
Use the existing function `naive_from_unix`

Reply:
I looked at pub  naive_from_unix(time: i64) and it uses seconds, not ms. My intention was to be client-friendly with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now which uses milliseconds. And  it further gives a consistent result because it's entirely possible get many posts in the same second... so I'm not sure what to do here with submitting a new commit.